test/ui-test/src/screens/StatusWalletScreen.py

```diff
@@ -216,3 +221,14 @@ def verify_saved_address_doesnt_exist(self, name: str):
 
     def verify_transaction(self):
         print("TODO: fix notification and ensure there is one")
+        
+    def verify_collectibles_exist(self, account_name: str):
+        tabbar = get_obj(MainWalletScreen.RIGHT_SIDE_TABBAR.value)
+        click_obj(tabbar.itemAt(1))
+        collectionsRepeater = get_obj(CollectiblesView.COLLECTIONS_REPEATER.value)
+        if(collectionsRepeater.count > 0):
+            collectionsRepeater.itemAt(0).expanded = True
+        collectiblesRepeater = get_obj(CollectiblesView.COLLECTIBLES_REPEATER.value)
+        if(collectionsRepeater.count > 0):
+            return True
+        return False           
```

Comment:
I think you need to do an actual verify here, just returning `False` would **not** trigger a test fail as far as I know.

Reply:
done :)